src/tour.js

```diff
@@ -38,24 +48,25 @@ disasterSearchTour.addStep('enter-search', {
       Start here by typing in a FEMA disaster ID.
       </p>
       <p>
-      These IDs follow the format “DR‐4272‐TX” But you can also type “4272”.
+      These IDs follow the format "DR‐4272‐TX" But you can also type "4272".
```

Comment:
I'm curious—why did you move from quotation marks to double primes (“straight quotes”) throughout tour.js?